qmlti/notify.php

```diff
@@ -52,9 +50,7 @@
     $outcome->setValue($score);
     $outcome->setResultID($report_id);
     $outcome->type = 'percentage';
-    if (!$resource_link->doOutcomesService(LTI_Resource_Link::EXT_WRITE, $outcome)) {
-      error_log("Error saving outcome of {$score} for {$result_id}");
-    } else {
+    if ($resource_link->doOutcomesService(LTI_Resource_Link::EXT_WRITE, $outcome)) {
       error_log("Successfully passed outcome of {$score} for {$result_id}");
       $outcome->saveToResult($consumer, $resource_link, $participant);
     }
```

Comment:
What do we do on error? shouldn't we throw an error instead of silently continuing?